src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export interface Tab {
+  id: string,
+  title: string,
+  content: string
+}
+
+interface TabsProps {
+  tabs: Tab[],
+  selectedTabId: string
+  setSelectedTabId: (selectedTabId:string) => void;
+}
+
+export const Tabs = ({ tabs, selectedTabId, setSelectedTabId }: TabsProps) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={classNames(
+                selectedTabId === tab.id ? 'is-active' : '',
+              )}
+              key={tab.id}
+              data-cy="Tab"
```

Comment:
be careful
![image](https://user-images.githubusercontent.com/98764641/227890361-24f828c5-82b0-4df1-bd8c-50a914c99dc5.png)